meta-mbl-distro/recipes-core/busybox/busybox/busybox_%.bbappend

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2019 Arm Limited and Contributors. All rights reserved.
+#
+# SPDX-License-Identifier: MIT
+
+# Add logrotate config file
+MBL_LOGROTATE_CONFIG_LOG_NAMES = "syslog"
+MBL_LOGROTATE_CONFIG_LOG_PATH[syslog] = "/var/log/messages"
+MBL_LOGROTATE_CONFIG_SIZE[syslog] = "1M"
+MBL_LOGROTATE_CONFIG_ROTATE[syslog] = "4"
+inherit mbl-logrotate-config
+
+FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
+
+# if_updown_ext_dhcp.cfg unsets CONFIG_FEATURE_IFUPDOWN_EXTERNAL_DHCP.
+# This means busybox ifup/ifdown will ignore an external managers such as ConnMan.
+SRC_URI += "file://if_updown_ext_dhcp.cfg"
+
+# Enable dd ibs and obs support so that it is possible to run the Qualcomm
+# firmware package.
+SRC_URI += "file://dd_ibs_obs.cfg"
```

Comment:
This looks like a BSP thing. Do we need to split this up so that this line ends up in one of the freescale*-mbl layers?

Reply:
In this case we will have to move the `dd_ibs_obs` to meta-freescale-mbl since it is used by `imx8mmevk-mbl`, `imx7d-pico-mbl` and `imx6ul-pico-mbl`. Do you agree?

Reply:
I don't know about the details, I'll take your word for it.

Reply:
This is basically for the binary release where we don't distribute the qca9733 firmware binaries.
It seems that the installer in the target uses the dd to install the binaries. I don't understand why but according to @niej this option is mandatory.

Reply:
fixed in 81dc088

Reply:
Thanks. Resolving.